assets/src/components/v2/departures/header.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useMemo } from "react";
+import cx from "classnames";
+import { imagePath } from "Util/util";
+
+type CardinalDirection = "n" | "ne" | "e" | "se" | "s" | "sw" | "w" | "nw";
+
+interface Props {
+  title: string | null;
+  direction: CardinalDirection | null;
+}
+
+const DirectionArrow = ({ direction }: { direction: CardinalDirection }) => {
+  const src = useMemo(() => imagePath("arrow-45.svg"), []);
+  return (
+    <img
+      src={src}
+      data-direction={direction}
+      className={cx("direction-arrow", {
+        "rotate-45": direction === "n",
+        "rotate-90": direction === "ne",
+        "rotate-135": direction === "e",
+        "rotate-180": direction === "se",
+        "rotate-225": direction === "s",
+        "rotate-270": direction === "sw",
+        "rotate-315": direction === "w",
+      })}
+    />
+  );
+};
```

Comment:
**note:** There is already a [similar component for "Solari"](https://github.com/mbta/screens/blob/main/assets/src/components/solari/arrow.tsx); however, while that component uses both `arrow.svg` and `arrow-45.svg`, this only uses `arrow-45.svg`. This produces slightly larger N, E, S, and W arrows (that are the same size as the oblique arrows but with a larger bounding box). This matches the [designs in Figma](https://www.figma.com/design/uyXu91sE66baBZMg81XOmk/ScreenKit?node-id=1212-2806&t=AF8TMdu5kySSqM5Z-4).

<img width="441" alt="Screenshot 2024-05-22 at 8 32 56 AM" src="https://github.com/mbta/screens/assets/1699281/862e4012-50d3-4ce5-8aba-25a1c269b60f">
<img width="221" alt="image" src="https://github.com/mbta/screens/assets/1699281/c184546a-76e0-4c75-8ea0-ee65f52ad160">